packages/driver/cypress/integration/e2e/multi-domain/multi_domain_validation_specs.ts

```diff
@@ -31,7 +31,7 @@ describe('multi-domain', () => {
       })
     })
 
-    it('succeeds on a unicode domain', () => {
+    it('succeeds on a unicode origin', () => {
```

Comment:
Since this is not a complete origin:
```suggestion
    it('succeeds on a unicode domain', () => {
```

Reply:
nice catch, thanks!